scripts/sysman/device.yml

```diff
@@ -355,6 +355,46 @@ returns:
     - $X_RESULT_ERROR_INVALID_SIZE:
         - "The provided value of pCount is not big enough to store information about all the processes currently attached to the device."
 
+--- #--------------------------------------------------------------------------
+type: struct
```

Comment:
for any new structs, please remember to add an enum in `common.yml`

Reply:
Missed it, Added now.